### PR TITLE
HitDumper for Commissioning

### PR DESCRIPTION
**Pull request by marcodeltutto**
This PR:
- moves the `HitDumper` module from the `AnalysisTree/` directory to its own directory called `Commissioning/`;
- incorporates a new feature to the `HitDumper` module to dump the whole waveform to check for transparency (feature implemented by Gavin Menning).

### CODE REVIEW

**Review by mstancar — APPROVED** [review]
Looks good!

**Comment by pgreen135** [comment]
was this intentionally closed?

**Comment by mstancar** [comment]
Probably not . . .I might have clicked the wrong thing when I was approving

From: Patrick Green ***@***.***>
Reply-To: SBNSoftware/sbndcode ***@***.***>
Date: Monday, March 22, 2021 at 5:19 PM
To: SBNSoftware/sbndcode ***@***.***>
Cc: Michelle D Stancari ***@***.***>, State change ***@***.***>
Subject: Re: [SBNSoftware/sbndcode] HitDumper for Commissioning (#91)


was this intentionally closed?

File: sbndcode/Commissioning/HitDumper_module.cc [hunk 1]
```diff
@@ -142,6 +146,13 @@ class Hitdumper : public art::EDAnalyzer {
   double _hit_charge[kMaxHits];        ///< Hit charge
   double _hit_ph[kMaxHits];            ///< Hit ph?
   double _hit_width[kMaxHits];         ///< Hit width
+  double _hit_full_integral[kMaxHits]; ///< Hit charge integral
+  int    _waveform_number[kMaxHits*5001];   ///< Number for each waveform, to allow for searching
+  double _adc_on_wire[kMaxHits*5001];       ///< ADC on wire to draw waveform
+  int    _time_for_waveform[kMaxHits*5001]; ///<Time for waveform to plot
```
Comment (absolution1) [inline sbndcode/Commissioning/HitDumper_module.cc:152]: Hey @marcodeltutto 
You might save yourself a modest amount of anguish down the line if you defined the array size as its own constant expression:

`constexpr int kMaxWaveforms = kMaxHits*5001;`

I dunno if you'd ever need to change the size of these arrays but the above change would mean only one point of maintenance in if you did need to.

**Pull request merged**